Add Project Template pages descriptions (#4692)
sourcegraph/about: handbook/product/design/documents_templates/index.md

## Figma

[Helpers & Tooling](https://www.figma.com/files/project/12944215/Helpers-%26-Tooling?fuid=872030793297791611) project includes:

> - [Project Template](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/Project-Template), which you can duplicate and use for all new Figma files. It includes some pages we’ve found work well for supporting async work and Figma files as a source of truth:

Reviewer: ```suggestion
- [Project Template](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/Project-Template), which you can duplicate to start new Figma files. This template is pre-structured with pages that support async work and maintain Figma files as a source of truth:
```

>   - [Cover:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=0%3A1) a project thumbnail used to recognising the files easier while browsing. Please, use the cover symbol for changing the stages (draft, in progress, done, deprecated) as the project progresses.

Reviewer: ```suggestion
  - [Cover:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=0%3A1) a project thumbnail used to easily and consistently identify files in Figma. Use the underlying cover symbol to change the stages (draft, in progress, done, deprecated) as the project progresses.
```

  - [Master:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=246%3A11) used for final, usually high-fidelity, mockups. The template showcased on this page can be used for grouping designs in bigger projects.

>   - [Prototypes:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=262%3A11) used to separate interactive prototypes mockups that may include stages and variants that designers don't want to include in the main design pages. These prototypes can be actively used for usability testing. Please, be careful when making any changes.

Reviewer: ```suggestion
  - [Prototypes:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=262%3A11) used to separate mockups from prototyped mockups that may include stages and variants that designers don't want to include in the main design pages. These prototypes can be actively used for usability testing, so be cautious in making changes to mockups in this page. 
```

>   - [Wireframes:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=103%3A3) used for the low-fidelity mockups.

Reviewer: ```suggestion
  - [Wireframes:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=103%3A3) used for the low-fidelity mockups. Designers can import sketches from Excalidraw into this page to centralize discussions.
```

Author: I wouldn't like us to suggest that Excalidraw sketches should only be used here

>   - [Buddy review:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=260%3A11) used for reviewing the in-progress work by other team memebrs or designers.

Reviewer: ```suggestion
  - [Buddy review:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=260%3A11) used for reviewing in-progress work with other team members or designers. Buddy reviews are often used when a designer would like feedback from an individual(s) before sharing with a larger group. 
```

>   - [Buddy review:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=260%3A11) used for reviewing the in-progress work by other team memebrs or designers. Buddy reviews are often used when a designer would like feedback from an individual(s) before sharing with a larger group.

Reviewer 2: ```suggestion
  - [Buddy review:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=260%3A11) used for reviewing the in-progress work by other team members or designers. Buddy reviews are often used when a designer would like feedback from an individual(s) before sharing with a larger group. Using the buddy review section of the project is helpful for simplifying review for others, because it focuses on just the elements for review.
```

>   - [Playgroud:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=246%3A12) a place for the designer to store the exploratory or in-progress mockups that don't need to be reviewed.

Reviewer: ```suggestion
  - [Playground:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=246%3A12) a place for the designer to store exploratory or in-progress mockups that don't need to be reviewed.
```

>   - [Playgroud:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=246%3A12) a place for the designer to store exploratory or in-progress mockups that don't need to be reviewed.

Reviewer 2: ```suggestion
  - [Playground:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=246%3A12) a place for the designer to store exploratory or in-progress mockups that don't need to be reviewed.
```

>   - [Redlines:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=291%3A227) used for optional detailed measurements. Use redlines only when absolutely necessary. It is not a recommended handoff method for the whole project.

Reviewer 2: ```suggestion
  - [Redlines:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=291%3A227) used for optional detailed measurements. Note that Figma has built-in redlines that should be used during implementation—use manual redlines only when absolutely necessary. It is not a recommended handoff method for the whole project.
```

>   - [Responsive layouts:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=302%3A227) used for XS-L mockups based on [our grids](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/?node-id=914%3A0). They apply not only to mobile devices. It's a common use case to view Soucegraph in 1/2 or 1/3 browser window on desktop as well.

Reviewer 2: ```suggestion
  - [Responsive layouts:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=302%3A227) used for XS-L mockups based on [our grids](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/?node-id=914%3A0). Responsive layouts are used not only for mobile devices, but for when viewing Sourcegraph in 1/2– or 1/3-width browser windows on desktops devices.
```
